apps/meteor/ee/app/livechat-enterprise/server/api/lib/outbound.ts

```diff
@@ -0,0 +1,33 @@
+import type { IOutboundMessageProviders } from '@rocket.chat/apps-engine/definition/outboundComunication';
+import type { IOutboundProvider } from '@rocket.chat/core-typings';
+
+import { OutboundMessageProvider } from '../../../../../../server/lib/OutboundMessageProvider';
+
+export class OutboundMessageProviderService {
+	private readonly provider: OutboundMessageProvider;
+
+	constructor() {
+		this.provider = new OutboundMessageProvider();
+	}
+
+	private isProviderValid(type?: string): type is 'phone' | 'email' {
```

Comment:
```suggestion
	private isProviderValid(type?: string): type is ValidProviderType {
```

And then we do a 
```
type ValidProviderType = 'phone' | 'email'
```

in case we don't have this type somewhere already. wdyt?